addons/survey/controllers/survey_session_manage.py

```diff
@@ -13,17 +13,31 @@
 
 
 class UserInputSession(http.Controller):
-    def _fetch_from_token(self, survey_token):
-        """ Check that given survey_token matches a survey 'access_token'.
+
+    # DANE: searching on survey_token is depreciated and it will be removed in v19/20.
+    # Searching only have to be done on survey_id.
```

Comment:
depreciate is the accounting term :p
```suggestion
    # DANE: searching on survey_token is deprecated and it will be removed in v19/20.
```